lib/commands/list-private.js

```diff
@@ -17,7 +17,8 @@ function formatChannel (chan) {
 
 function displayGroupList (filter, groups) {
   const str = groups.reduce((acc, group) => (
-    `${acc}  ~ ${formatChannel(group)}\n`
+    // Subtracting 1 from `group.members.length` to exclude Wheelie.
+    `${acc}  ~ ${formatChannel(group)} (${group.members.length - 1})\n`
```

Comment:
Legit. I'd probably have filtered out the entry and then counted, but c'est l'code.